### ISSUE TITLE
Direct dependencies

### ISSUE THREAD

**Issue by njlr** [LoopPerfect/buckaroo#63]
Depend on recipes without a cook-book. For example: 
```
  "dependencies":  {
    direct: {
      "git@github.com:ericniebler/range-v3.git#256f2b8754159472fe48ab3c26f51cc9f1c7ccb5": {
        "name": "range-v3"
      }
    }
  }
```

**Comment by nikhedonia** [comment]
Works in release/1.2.0

**Comment by njlr** [comment]
Implemented in 1.2.0.
